test/core.jl

```diff
@@ -23,9 +23,8 @@
     end
 
     @testset "Hybrid NB" begin
-
         # a test to check that HybridNB successfully replaces KernelNB
-        m1 = HybridNB(y, 2)
+        m1 = HybridNB(y, ["c1", "c2"])
```

Comment:
It was supporting both but than I ran into a problem with the HDF5: the symbols were converted to strings which can result in some inconsistencies. The only way I could come up with keeping that information was to actually write the type of the names into the file and convert it, but it seems a bit clunky. Do you have any suggestions?